app/boards/arm/sparkfun_micromod_nrf52840/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+  COMMAND ${PYTHON_EXECUTABLE} ${ZEPHYR_BASE}/../tools/uf2/utils/uf2conv.py
+  -c 
+  -b 0x26000
+  -f 0xADA52840
+  -o ${PROJECT_BINARY_DIR}/${CONFIG_KERNEL_BIN_NAME}.uf2
+  ${PROJECT_BINARY_DIR}/${CONFIG_KERNEL_BIN_NAME}.bin
+)
```

Comment:
No longer needed as of recent changes, see: https://github.com/zmkfirmware/zmk/commit/4eb8f8cd23df49d366034da674293bef01c2aacd